usaspending_api/references/models.py

```diff
@@ -462,122 +462,117 @@ class LegalEntity(DataSourceTrackedModel):
 
     individuals
     '''
-    business_categories = ArrayField(models.TextField(), default=list)
+    business_categories = ArrayField(models.TextField(), default=list, null=True)
```

Comment:
older data?

I have firsthand issues with this field because testing scripts with a local (orm-defined) db would fail but the staging and production runs were fine. The default values are most likely a recent change

Reply:
Well, whatever the cause, this has been fixed.  Hopefully whatever caused the nulls has been resolved or surfaces when we test on staging.

Reply:
Also, you'll want to pull the latest version of the DB script from Jira.  I added code in there to backfill.